variables.tf

```diff
@@ -16,6 +16,9 @@ variable "repos" {
     })))
     workflow = optional(string)
     ######### Helpers #########
+    infracost = optional(object({
```

Comment:
I'm thinking if that shouldn't be computed somehow - for example. if `infracost` is enabled in `workflow`, we enable it globally.

I can see misconfigurations here - when user enables infracost in workflow but forgets to do it in `repos`

Reply:
I agree that it should be computable for most cases but maybe with possibility to override?
for example:
* repo config allows to override `workflow`
* current (default) workflow doesn't use infracost
* user picks workflow (it could be server defined) that uses `infracost`

btw maybe it's not a reasonable scenario? 🤔 
(I thought a little bit about this, but didn't come to a conclusion)

Reply:
I can imagine a use-case when override would be needed, so it is porbably a good idea